data/dapps/dtrinity.json

```diff
@@ -1,5 +1,7 @@
 {
-  "alias": "dtrinity",
-  "name": "dTrinity",
+  "aliases": {
+    "dtrinity": ["fraxtal"]
+  },
+  "name": "dTRINITY",
```

Comment:
This one seems to have changed since I looked at it

Reply:
Yeah, as I mentioned - I've renamed them according to Defillama. But they're using this capitalization also on their homepage.